webserver/src/main/java/io/kestra/webserver/controllers/api/ExecutionController.java

```diff
@@ -794,6 +794,27 @@ public Execution restart(
         return restart;
     }
 
+    @ExecuteOn(TaskExecutors.IO)
+    @Post(uri = "/{executionId}/restart-with-input")
```

Comment:
Those endpoint (duplicated of the original one for now) are not necessary, just need to enhance the existing one

same apply for the replay endpoint